lib/modules/datasource/custom/schema.ts

```diff
@@ -1,12 +1,13 @@
 import { z } from 'zod';
+import { TimestampSchema } from '../../../util/timestamp';
 
 export const ReleaseResultZodSchema = z.object({
   releases: z.array(
     z
       .object({
         version: z.string(),
         isDeprecated: z.boolean().optional(),
-        releaseTimestamp: z.string().optional(),
+        releaseTimestamp: TimestampSchema.nullable().catch(null),
```

Comment:
shouldn't we create a shortcut for this? So we've less duplicate code?

```ts
export type TimestampSchemaSafe = TimestampSchema.nullable().catch(null);
export type NullableTimestampSchema = TimestampSchema.nullable().catch(null);
```

or even make `TimestampSchema` nullable and catch to null?

Reply:
Yes, I think just make it nullable is reasonable and is consistent with `asTimetamp` return type